src/target/stm32f4.c

```diff
@@ -233,6 +235,7 @@ bool stm32f4_probe(target *t)
 		t->driver = stm32f4_get_chip_name(idcode);
 		t->attach = stm32f4_attach;
 		target_add_commands(t, stm32f4_cmd_list, t->driver);
+		target_mem_write32(t, DBGMCU_CR, 7);
```

Comment:
nit pick, would be nice to have this be defines to make it a bit more clear what's going on... This is in `stm32l4.c` but could be moved to a common spot:
```c
#define DBGMCU_CR_DBG_SLEEP		(0x1U << 0U)
#define DBGMCU_CR_DBG_STOP		(0x1U << 1U)
#define DBGMCU_CR_DBG_STANDBY	(0x1U << 2U)
```

and then you could do:
```suggestion
		target_mem_write32(t, DBGMCU_CR, DBGMCU_CR_DBG_SLEEP | DBGMCU_CR_DBG_STOP | DBGMCU_CR_DBG_STANDBY);
```

I think that's what this is doing, right?